lib/template-bundler.js

```diff
@@ -15,7 +15,7 @@ export function bundleTemplate(pattern, fileName, _opts) {
 
   jspm.setPackagePath(options.packagePath);
 
-  var builder = new jspm.Builder();
+  var builder = new jspm.Builder(options.builder);
```

Comment:
As, we have `jspm.setPackagePath(options.packagePath);` already set here, the `bundler` will pickup all correct `directories` including `baseURL` from `package.json`. What other benefits are we getting from passing `options.builder`  to the `Builder` constructor?